includes/definitions/fiberhome-switch.yaml

```diff
@@ -0,0 +1,19 @@
+os: fiberhome-switch
+text: 'FiberHome Switch'
+type: network
+icon: fiberhome
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.3807.1.582815
+        - .1.3.6.1.4.1.3807.1.281802
+        - .1.3.6.1.4.1.3807.1.481005
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
```

Comment:
As you have sensors here you can also add them to this.

I.e:

```
includes/definitions/powerlogic.yaml:    - { graph: sensor_voltage, text: 'Voltage' }
includes/definitions/powerlogic.yaml:    - { graph: sensor_current, text: 'Current' }
includes/definitions/powerlogic.yaml:    - { graph: sensor_power, text: 'Power' }
includes/definitions/powerlogic.yaml:    - { graph: sensor_frequency, text: 'Frequency' }
```

Reply:
Yes I am aware. The only issue is that some variant devices don't have the sensors and then you get that ugly empty box. Is there a way to suppress that mouse over instruction for those?